category_page.html

```diff
@@ -21,9 +21,12 @@
         class="logo"
       />
       <nav class="main-navbar">
+        <h1 class="shop-name">Shopapa</h1>
         <ul>
           <li class="nav-link">
-            <a href="#">Login</a>
+            <button type="button" class="login-button">
```

Comment:
We usually use either `button` or `anchor`.  
In this file it seems that most of the anchors could be removed in favour of only using a button.